### PR TITLE
Bug fix/knite 1113 style issues

### PR DESCRIPTION
**Pull request by rohanrjohn**

### CODE REVIEW

File: src/ui/PopUpManager.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import {clamp} from './clamp';
 import {fromHTMlElement, fromXY, isIntersected} from './rects';
 import type {PositionHandler} from './PopUpPosition';
 import type {Rect} from './rects';
+/* eslint-disable */
```
Comment (Tezra) [inline src/ui/PopUpManager.ts:5]: No disable

File: src/ui/PopUpManager.ts [hunk 2]
```diff
@@ -217,13 +179,10 @@ export class PopUpManager {
         }
       }
     }
-    /*eslint no-constant-condition: ["error", { "checkLoops": false }]*/
+
     while (true) {
```
Comment (Tezra) [inline src/ui/PopUpManager.ts:183]: why isn't this a do-while loop?

**Review by Tezra — CHANGES_REQUESTED** [review]

**Comment by Tezra** [comment]
@rohanrjohn there are some sonar issues that need to be fixed. In particular we really need to fix the deprecated react render issues. The rest of the issues look like they should be really quick to fix.

**Review by Tezra — CHANGES_REQUESTED** [review]
Linting is failing.

After failing the build I also merged in the weekly dep updates.

**Comment by rohanrjohn** [comment]
Closing pull request as the functionality seems to be breaking after the sonar fixes.

**Pull request closed without merging**